src/codecs/md/index.ts

```diff
@@ -829,6 +838,48 @@ function encodeDelete(delet: stencila.Delete): MDAST.Delete {
   }
 }
 
+/**
+ * Decode a MDAST `sub` node to a Stencila `Subscript` node.
+ */
+const decodeSubscript = (sub: MDAST.Parent): stencila.Subscript => {
+  return stencila.subscript(
+    sub.children.map(node => decodePhrasingContent(node as MDAST.PhrasingContent))
+  )
+}
+
+/**
+ * Encode a Stencila `Subscript` as a MDAST `text` node with surrounding tildes.
+ *
+ * This assumes that there is only `string`s in the `content` of the subscript.
+ */
+const encodeSubscript = (sub: stencila.Subscript): MDAST.Text => {
+  return {
+    type: 'text',
+    value: `~${sub.content.map(item => item !== null ? item.toString() : '').join()}~`
```

Comment:
Think it might be more resilient if we use `JSON.stringify(item)` here, as most non-string types will throw here.
Also, the documentation says that we assume there is only `string` in the `content`, so should we be filtering out non-`string` values instead?

Reply:
Yes, that is a better approach: filter out, and warn of, non-`string` content